src/plugins/contextMenu/predefinedItems/rowBelow.js

```diff
@@ -35,7 +35,7 @@ export default function rowBelowItem() {
         return true;
       }
 
-      return this.selection.isSelectedByColumnHeader() || this.countRows() >= this.getSettings().maxRows;
+      return this.selection.isSelectedByColumnHeader() || this.selection.isSelectedByCorner() || this.countRows() >= this.getSettings().maxRows;
```

Comment:
The line exceeds 120 characters. Additionally, I didn't found any test about disability the "Insert row below" option in the context menu when the corner is clicked. Can I ask you to fix that?